compiler-plugin-test/src/test/java/io/ballerina/stdlib/persist/compiler/CompilerPluginTest.java

```diff
@@ -323,22 +323,6 @@ public void testInvalidInitialisation3() {
                 });
     }
 
-    @Test
-    public void testUnSupportedFeature() {
-        List<Diagnostic> errorDiagnosticsList = getDiagnostic("package_17", 1);
```

Comment:
Could you delete files under `package_17` too?

Reply:
Deleted in [c89415a](https://github.com/ballerina-platform/module-ballerina-persist/pull/82/commits/c89415aa14bac951b1154e7fef41837561cb3e75)